libs/Microsoft.MixedReality.WebRTC.Native/include/interop_api.h

```diff
@@ -70,23 +70,112 @@ MRS_API void MRS_CALL mrsCloseEnum(mrsEnumHandle* handleRef) noexcept;
 // Interop
 //
 
+struct mrsAudioTransceiverConfig;
+struct mrsVideoTransceiverConfig;
+struct mrsRemoteAudioTrackConfig;
+struct mrsRemoteVideoTrackConfig;
 struct mrsDataChannelConfig;
 struct mrsDataChannelCallbacks;
 
+/// Opaque handle to a native PeerConnection C++ object.
+using PeerConnectionHandle = void*;
+
+/// Opaque handle to a native MediaTrack C++ object.
+using MediaTrackHandle = void*;
+
+/// Opaque handle to a native AudioTransceiver C++ object.
+using AudioTransceiverHandle = void*;
+
+/// Opaque handle to a native VideoTransceiver C++ object.
+using VideoTransceiverHandle = void*;
+
+/// Opaque handle to a native LocalAudioTrack C++ object.
+using LocalAudioTrackHandle = void*;
+
+/// Opaque handle to a native LocalVideoTrack C++ object.
+using LocalVideoTrackHandle = void*;
+
+/// Opaque handle to a native RemoteAudioTrack C++ object.
+using RemoteAudioTrackHandle = void*;
+
+/// Opaque handle to a native RemoteVideoTrack C++ object.
+using RemoteVideoTrackHandle = void*;
+
+/// Opaque handle to a native DataChannel C++ object.
+using DataChannelHandle = void*;
+
+/// Opaque handle to a native ExternalVideoTrackSource C++ object.
+using ExternalVideoTrackSourceHandle = void*;
```

Comment:
Should either have a prefix or be inside a namespace.

Reply:
Fixed.